geode-membership/build.gradle

```diff
@@ -66,7 +65,9 @@ dependencies {
     testImplementation('com.tngtech.archunit:archunit-junit4')
 
     integrationTestImplementation('com.tngtech.archunit:archunit-junit4')
-  
+
+    concurrencyTestImplementation sourceSets.integrationTest.output
```

Comment:
This doesn't feel great. What is the dependency here?

Reply:
Some "framework" type classes shared between the two. The "correct" approach is to create a shared sub module but it's one class. We could iterate on it. The goal wasn't to solve all the worlds problems today.